.claude/commands/kb/fixture/adapter/request.md

```
@@ -0,0 +1,89 @@
+**템플릿 구조**:
+```java
+package com.ryuqq.fixture.adapter.rest;
+
+import com.ryuqq.adapter.in.rest.dto.[RequestName];
+
+/**
+ * [RequestName] Request DTO Test Fixture
+ *
+ * @author development-team
+ * @since 1.0.0
+ */
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)